flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/sink/BaseDeltaTaskWriter.java

```diff
@@ -103,5 +103,10 @@ public void write(RowData row) throws IOException {
     protected StructLike asStructLike(RowData data) {
       return wrapper.wrap(data);
     }
+
+    @Override
+    protected StructLike asStructLikeKey(RowData data) {
+      throw new UnsupportedOperationException("Not implemented for Flink 1.13 during PR review");
```

Comment:
@rdblue do you want me to implement these for older Flink versions in this PR or handle them in a follow up?

Reply:
I'll do them in a follow up. These are not used in any of the 1.12 or 1.13 code paths, just necessary for compilation. So there's no commit that contains a solution that's half fixed.